src/amo/reducers/users.js

```diff
@@ -176,14 +216,17 @@ export const getCurrentUser = (users: UsersStateType) => {
     return null;
   }
 
-  if (!users.byID[users.currentUserID]) {
-    throw new Error(
-      'currentUserID is defined but no matching user found in users state.');
-  }
+  invariant(users.byID[users.currentUserID],
+    'currentUserID is defined but no matching user found in users state.');
 
   return users.byID[users.currentUserID];
 };
 
+export const getUserById = (users: UsersStateType, userId: number) => {
```

Comment:
This isn't used anywhere. Is it something the will be needed for the saga?

Reply:
Looks like I was *going* to use it in here and didn't. I'll either update the code to use it or remove it.